firmware/esp32/main.ino

```diff
@@ -387,6 +387,9 @@ void setup()
 {
   Serial.begin(115200);
   delay(200);
+  
+  setCpuFrequencyMhz(80); //Set CPU clock to 80MHz fo example
```

Comment:
I'll change to `setCpuFrequencyMhz(CLOCK_SPEED)` and then in `config.h` add
`#define CLOCK_SPEED 240`